src/SFA.DAS.EmployerAccounts.Web/ViewModels/AccountDashboardViewModel.cs

```diff
@@ -37,5 +37,8 @@ public class AccountDashboardViewModel
         public bool ShowReservations => AccountViewModel?.Organisations?.FirstOrDefault()?.Reservations?.Count > 0;
         public bool HasSingleProvider => AccountViewModel?.Providers?.Count == 1;
         public bool HasMultipleProviders => AccountViewModel?.Providers?.Count > 1;
+        public bool HasSingleDraftVacancy => AccountViewModel?.Vacancies?.Count(v => v.Status == VacancyStatus.Draft) == 1;
```

Comment:
The `Account` returned from `GetAccount()` now returns 2 properties, `VacancyCardinality` and `SingleVacancy`. So to check for a single vacancy of a particular status, look for `VacancyCardinality` set to `Cardinality.One`, and then check the status of the `SingleVacancy`.